app/templates/admin/createEvent.html

```diff
@@ -129,7 +129,7 @@ <h1 id="title">{{page_title}}</h1>
                 {%else%}
                   {%set end_time = "12:00" %}
                 {% endif %}
-                  <input autocomplete="off" type="time" class="form-control timepicker readonly" value="{{end_time}}" name="timeEnd" aria-describedby="timeIcon2" placeholder="Pick an end time" onchange="update(this)" id=endTime required />
+                  <input autocomplete="off" type="time" class="form-control timepicker" value="{{end_time}}" name="timeEnd" aria-describedby="timeIcon2" placeholder="Pick an end time" onchange="update(this)" id=endTime required />
```

Comment:
Nice, this works but, if a user went to type something like 1:30, they would have to manually move over to the minute section with either a mouse click or a right arrow key. It would be nice to catch the : and move to the next input (the minutes) so the user doesn't have to manually move themselves.

Reply:
That's right. But I am a little confused. The user has to input two digits on the left and on the right of a colon symbol (:). That's why it doesn't let the user input 1:30, it has to be 01:30. Wouldn't removing the option of choosing two digits on the left result in the user not being able to choose time after 9, meaning 10:00, 11:00, and so on?